SEMANTICS.md

```diff
@@ -107,15 +134,44 @@ Level 1 supports:
 
 ## Level 2
 
-### Commands
+### Commands and types
 
 Level 2 supports:
-* `print`
-* `ask`
-* `forward`
-* `turn`
-* `is`
-* `sleep`
+
+<table>
+<thead>
+  <tr>
+    <th>Command</th>
+    <th>Types</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>print</td>
+    <td>integer | string</td>
+  </tr>
+  <tr>
+    <td>ask</td>
+    <td>integer | string</td>
+  </tr>
+  <tr>
+    <td>forward</td>
+    <td>integer</td>
+  </tr>
+  <tr>
+    <td>turn</td>
+    <td>integer</td>
+  </tr>
+  <tr>
+    <td>is</td>
```

Comment:
type signature of is should have two types I think, left and right? Left of the is we can have only valid variable names (not integers for example or symbols) and right we can have anything.

Reply:
valid variable names would be strings, I think?